no-default-features: can't find crate deflate

For:
```toml
[dependencies.png]
default-features = false
version = "0.14"
```

I get:

```
   Compiling png v0.14.0
error[E0463]: can't find crate for `deflate`
 --> /home/yeet/.cargo/registry/src/github.com-1ecc6299db9ec823/png-0.14.0/src/common.rs:2:1
  |
2 | extern crate deflate;
  | ^^^^^^^^^^^^^^^^^^^^^ can't find crate

error: aborting due to previous error
```

Seems to have been added by 477679295b20a6a7d6e9713b8e8efd720643ca79, and definitely looks incorrect to me; `common.rs` says "shared", so can't have non-default stuff like that in it?

(I'm on modern Ubuntu, recent nightly Rust. Almost certainly irrelevant.)

--- a/.travis.yml
+++ b/.travis.yml
@@ -2,6 +2,8 @@ language: rust
 matrix:
   include:
     - rust: stable
+    - rust: stable
+      env: FLAGS="--no-default-features"
     - rust: nightly
     - rust: nightly
       env: FLAGS="-Z minimal-versions"

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -1,6 +1,6 @@
 [package]
 name = "png"
-version = "0.14.0"
+version = "0.14.1"
 license = "MIT/Apache-2.0"
 description = "PNG decoding and encoding library in pure Rust"
 authors = ["nwin <nwin@users.noreply.github.com>"]

